browser/ui/toolbar/brave_bookmark_context_menu_controller.cc

```diff
@@ -43,6 +45,12 @@ BraveBookmarkContextMenuController::BraveBookmarkContextMenuController(
   }
   AddBraveBookmarksSubmenu(profile);
   AddShowAllBookmarksButtonMenu();
+
+  if (bookmarks::BookmarkModel* model =
+          BookmarkModelFactory::GetForBrowserContext(profile)) {
+    is_other_bookmark_node_empty =
+        (model->other_node()->GetTotalNodeCount() == 1);
```

Comment:
ditto

Reply:
It'd be good to have comment why we're comparing 1 not 0. I guess it count other node itself, doesn't it? In that case, `other_node()->children().empty()` would be more readable.

Reply:
yeah! makes sense.

Reply:
> It'd be good to have comment why we're comparing 1 not 0. I guess it count other node itself, doesn't it? In that case, `other_node()->children().empty()` would be more readable.

Made this change. I guess with `other_node()->children().empty()` it is self-explanatory, no comments needed, right?

Reply:
Yup, I think so 👍